components/builder-api/src/config.rs

```diff
@@ -64,6 +66,7 @@ impl Default for Config {
             github_client_secret: DEV_GITHUB_CLIENT_SECRET.to_string(),
             ui_root: None,
             events_enabled: false, // TODO: change to default to true later
+            log_dir: "/tmp".to_string(),
```

Comment:
Potentially use something like [`std::env::temp_dir()`](https://doc.rust-lang.org/std/env/fn.temp_dir.html) which appears to do the Right Things™?